tests/playwright/utils/time.ts

```diff
@@ -0,0 +1,26 @@
+/** Format the given date as `MM/dd/yyyy`. Example: "12/30/2024" */
+const formatDateUS = (date: Date | number) => {
+  const formatter = new Intl.DateTimeFormat("en-US", {
+    year: "numeric",
+    month: "numeric",
+    day: "numeric",
+  });
+  const parts = formatter.formatToParts(date);
+  const getPart = (type: string) => parts.find((p) => p.type === type)!.value;
+  const yyyy = getPart("year");
+  const MM = getPart("month");
+  const dd = getPart("day");
+  return `${MM}/${dd}/${yyyy}`;
+};
+
+/** The length of one day, in milliseconds (ignoring DST and leap seconds) */
+const ONE_DAY = 24 * 60 * 60 * 1000;
+
+/** Yesterday's date, in local time, as MM/dd/yyyy. Example: "12/29/2024" */
+export const yesterday = () => formatDateUS(new Date().valueOf() - ONE_DAY);
```

Comment:
Do we ever use something like momentjs for stuff like this?
https://momentjs.com/docs/#/manipulating/

Reply:
Why, yes!
* A lot of our apps included moment as a dependency for a long time
* We gradually shifted to prefer date-fns
* We recently had a [raft of stories](https://jiraent.cms.gov/issues/?jql=text%20~%20momentjs%20ORDER%20BY%20updated%20DESC) to remove moment - there was a related security warning, and we took it as an opportunity to consolidate
* In almost all cases, I prefer to use JS built-ins over imported packages, so that's what I did here
* In the future I hope to revisit those places where we're still using date-fns, and replace them with Intl.DateTimeFormat or similar
* I think this is feasible because we don't, in general, have a very date-heavy application. There is very little calendar math in MDCT, and only one time zone conversion (which may be superfluous anyway - but don't tell Cameron I said that (yet)).